INV4/pallet-inv4/src/ipt.rs

```diff
@@ -56,29 +57,34 @@ impl<T: Config> Pallet<T> {
     ) -> DispatchResult {
         let owner = ensure_signed(owner)?;
 
+        // IP Set must exist for their to be a token
```

Comment:
```suggestion
        // IP Set must exist for there to be a token
```

Reply:
Been trying to figure out these 2 words for 26 years